src/components/CreateVmWizard/steps/Networking.js

```diff
@@ -297,7 +284,7 @@ class Networking extends React.Component {
           id: newId,
           name: nextNicName,
           deviceType: NIC_INTERFACE_DEFAULT,
-          vnicProfileId,
+          vnicProfileId: vnicEmptyProfileId,
```

Comment:
Just use a file constant like `EMPTY_ID` or `UNASSIGNED_ID` to match what is done in `NicEditor`.  Empty/unassigned will always be the default, no need to pass it through props.